src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Utilities/Element/BitElementDemo.razor.scss

```
@@ -0,0 +1,36 @@
+﻿// The examples render loose elements of every kind next to each other, so they need a container that puts a
+// consistent gap between them and keeps the inline ones (an anchor, a mark) from stretching across the row.
+.example-content {
+    gap: 0.5rem;
+    display: flex;
+    flex-flow: column;
+    align-items: flex-start;
+
+    // A rule is a block element with no content of its own, so in a column that sizes its items to their
+    // content it would come out zero pixels wide and stay invisible.
+    > ::deep hr {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)